<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Settings</class>
 <widget class="QWidget" name="Settings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>533</width>
    <height>406</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Show</string>
      </attribute>
      <widget class="QCheckBox" name="show_Biomes">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>200</y>
         <width>122</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>activate Biomes</string>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>128</width>
         <height>158</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QCheckBox" name="show_Bord_Case">
          <property name="text">
           <string>Bord Case</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="show_Perception">
          <property name="text">
           <string>Perception</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="show_Memory">
          <property name="text">
           <string>Memory</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="show_Nature">
          <property name="text">
           <string>Nature</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>181</x>
         <y>21</y>
         <width>133</width>
         <height>77</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QCheckBox" name="show_Minimap">
          <property name="text">
           <string>Minimap</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="show_Food_ProgressBar">
          <property name="text">
           <string>Food ProgressBar</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Controler</string>
      </attribute>
      <widget class="QWidget" name="layoutWidget_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>152</width>
         <height>104</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3"/>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Bobs</string>
      </attribute>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>152</width>
         <height>104</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2"/>
      </widget>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
     <property name="centerButtons">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Settings</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>47</x>
     <y>373</y>
    </hint>
    <hint type="destinationlabel">
     <x>244</x>
     <y>350</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Settings</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>169</x>
     <y>373</y>
    </hint>
    <hint type="destinationlabel">
     <x>468</x>
     <y>350</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Settings</receiver>
   <slot>update_Config()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>306</x>
     <y>392</y>
    </hint>
    <hint type="destinationlabel">
     <x>531</x>
     <y>367</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>update_Config()</slot>
 </slots>
</ui>
